src/libostree/ostree-async-progress.c

```diff
@@ -331,6 +331,8 @@ ostree_async_progress_set (OstreeAsyncProgress *self,
   if (self->dead)
     goto out;
 
+  changed = FALSE;
```

Comment:
I personally associate the assignment here more closely with the block below than with the actual function. Put another way, if there was a second block after this one which also uses `changed` (separately), then I wouldn’t want to rely on initialisation at declaration time for it.

But I’m not really fussed either way. I’ll change it if you want.